src/views/Home/Home.js

```diff
@@ -7,12 +7,16 @@ import './Home.css';
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
-
+  const [title, setTitle] = useState('afef');
+  const [subtitle, setSubtitle] = useState('afef');
+  const [font, setFont] = useState('comforter');
+  const [align, setAlign] = useState('');
+  const [text, setText] = useState('aff');
```

Comment:
It looks like you kept the default values in the our useState hook, always good to clean these up after you do some diagnosing, just an FYI. Nice work!